code/modules/admin/IsBanned.dm

```diff
@@ -4,8 +4,8 @@
 	if (!key || !address || !computer_id)
 		log_access("Failed Login (invalid data): [key] [address]-[computer_id]")
 		return list("reason"="invalid login data", "desc"="Error: Could not check ban status, Please try again. Error message: Your computer provided invalid or blank information to the server on connection (byond username, IP, and Computer ID.) Provided information for reference: Username:'[key]' IP:'[address]' Computer ID:'[computer_id]'. (If you continue to get this error, please restart byond or contact byond support.)")
-	
-	if (text2num(computer_id) == 2147483647) //this cid causes stickybans to go haywire 
+
+	if (text2num(computer_id) == 2147483647) //this cid causes stickybans to go haywire

```

Comment:
We are all aware that there's a VB GUI floating around that hooks Byond and changes the reported CID, right?